src/react-components/ui-root.js

```diff
@@ -635,34 +635,41 @@ class UIRoot extends Component {
       ) : null;
 
     const micPanel =
-      this.state.entryStep === ENTRY_STEPS.mic_grant || this.state.entryStep == ENTRY_STEPS.mic_granted ? (
+      this.state.entryStep === ENTRY_STEPS.mic_grant || this.state.entryStep === ENTRY_STEPS.mic_granted ? (
         <div className="mic-grant-panel">
-          <div className="mic-grant-panel__title">
```

Comment:
Mostly indentation differences below this line. I'd recommend the white space insensitive diff: https://github.com/mozilla/hubs/pull/290/files?w=1